### DISCUSSION TITLE
Super Robot Wars Games V, T X

### DISCUSSION THREAD

**Discussion by Berdugobugok** [skyline-emu/skyline#2211] [Game Requests]
We hope that the problem with the black screen after the battle and during the intermission menu will be fixed. This is the only problem this game have.

I hope our developers fix this. Thank you

**Comment by Lockondynam** [comment]
Yes please do

**Comment by Berdugobugok** [comment]
Bump thanks

**Comment by Berdugobugok** [comment]
Hello my beloved skyline developers. Im still waiting. versions after versions, still waiting. 🙂

**Comment by Berdugobugok** [comment]
Its already April version 61 and im still waiting Skyline Edge developers 👌
